### DISCUSSION TITLE
Some suggestions from official documentation

### DISCUSSION THREAD

**Discussion by zongyanglaobiao** [0xJacky/nginx-ui#658] [Q&A]
Hello, please forgive me for being rude and pointing out some issues with the documentation. The installation failed according to the Getting Started docker method in the documentation. The problem was that the documentation did not briefly describe the importance of the directory structure, because the log kept throwing errors, specifically, it could not find certain files/folders that were not in the documentation, and the page could not be seen when accessing the relevant ports. So I installed it according to the script method, but found that there was no download link for the relevant script? I think there is still room for improvement in the documentation.

**Comment by 0xJacky** [comment]
Hi, @zongyanglaobiao 

Thank you for raising this issue, and I’m truly sorry for the trouble the documentation caused you.

If you could provide the installation logs and a bit more detail on your setup environment (e.g., OS, Docker version, etc.), that would be a huge help in troubleshooting and improving the documentation.

Thanks again for your feedback! We’re looking forward to resolving this for you and improving the experience for future users.

Best regards,
Jacky

**Comment by felixzhang-glitch** [comment]
I also encountered a similar issue, with the error being a 404. After waiting for a while and retrying, it recovered. This part is likely due to the service initializing, but there was no prompt, and nothing was logged either. It completely relies on the user to guess, which is indeed not very user-friendly.

**Comment by 0xJacky** [comment]
> I also encountered a similar issue, with the error being a 404. After waiting for a while and retrying, it recovered. This part is likely due to the service initializing, but there was no prompt, and nothing was logged either. It completely relies on the user to guess, which is indeed not very user-friendly.

Hi @felixzhang-glitch, could you please provide us more detail like logs or screenshots?
